src/System.Private.CoreLib/src/System/Threading/ClrThreadPool.HillClimbing.cs

```diff
@@ -36,9 +72,9 @@ public enum StateOrTransition
             private readonly double _maxChangePerSecond;
             private readonly double _maxChangePerSample;
             private readonly int _maxThreadWaveMagnitude;
-            private readonly int _sampleIntervalLow;
+            private readonly int _sampleMsLow;
```

Comment:
Would be useful to retain "interval" in the name because that conveys what the value is (lower bound of sample interval length in ms), same for _sampleMsHigh and _currentSampleMs